tests/Type/WFS/Capabilities200Test.php

```diff
@@ -171,4 +171,27 @@ public function testSerialize(): void
         $capabilities->setVersion('2.0.0');
         $serializer = SerializerFactory::create();
     }
+
+    public function testMetadataURL(): void
+    {
+        $xml          = file_get_contents(FIXTURE_PATH . '/WFS/Capabilities_geoserver_pdok-20.xml');
+        $serializer   = SerializerFactory::create();
+        $capabilities = $serializer->deserialize($xml, Capabilities200::class, 'xml');
+        $featureTypes = $capabilities->getFeatureTypeList()->getFeatureTypes();
+        $this->assertCount(2, $featureTypes);
+        foreach ($featureTypes as $featureType) {
+            $this->assertCount(1, $featureType->getMetadataURLs());
+        }
+
+        $this->assertCount(1, $featureTypes['weggeg:weggegaantalrijbanen']->getMetadataURLs());
+        $this->assertCount(1, $featureTypes['weggeg:weggegaantalrijbanen']->getMetadataURLs());
```

Comment:
Is it required to check it for 2 times? maybe it would be  `$featureTypes['weggeg:weggegmaximumsnelheden']`.